docs/commands/json.mset.md

```diff
@@ -1 +1,60 @@
-Sets or updates the JSON value of one or more keys.
+Set or updates JSON values of one or more keys (multiple) for a given tripplet `key-path-value`
+
+JSON.MSET is atomic, so all given keys are set at once or not. It is not possible for clients to see that some of the keys were updated while others are unchanged.
+
+Important, JSON is a hierarquical structure if you change a value in specific path, subsequent nested values will be affected and may be unchanged
+
+
+[Examples](#examples)
+
+## Required arguments
+
+<details open><summary><code>key</code></summary>
+
+is key to modify.
+</details>
+
+<details open><summary><code>path</code></summary>
+
+is JSONPath to specify. Default is root `$`. For new Redis keys the `path` must be the root. For existing keys, when the entire `path` exists, the value that it contains is replaced with the `json` value. For existing keys, when the `path` exists, except for the last element, a new child is added with the `json` value.
```

Comment:
Default is root `$`
- How can we have a default? This is a required argument.

Reply:
It's optional